lib/fluent/plugin/in_tail.rb

```diff
@@ -357,6 +359,11 @@ def convert_line_to_event(line, es, tail_watcher)
             record[@path_key] ||= tail_watcher.path unless @path_key.nil?
             es.add(time, record)
           else
+            if @enable_catch_all
+              record = {'parse_fail' => line}
+              record[@path_key] ||= tail_watcher.path unless @path_key.nil?
+              es.add(Time.now.to_i, record)
```

Comment:
If you want this feature in v0.12, you also need to send a patch to v0.12 branch.